### ISSUE TITLE
Files too difficult to download from jaxb.dev.java.net

### ISSUE THREAD

**Issue by Tomas-Kraus** [eclipse-ee4j/jaxb-ri#746]
A user trying Download new releases faces almost insurmountable difficulties.

It appears that files are very difficult, sometimes impossible to download using
conventional means because:
1) on internet servers, the required ports (for download via a secure
connection) are not necessarily open.
2) the jaxb.dev.java.net server does not deliver a valid response because the
downloaded files are corrupted
3) conventional download clients do not necessarily handle secure connections well
4) secure downloads are not resumable
5) another server fisheye5.atlassian.com does not provide a valid content length
header

I have tried my best to get the files, including asking someone else (with a
broadband connection)
to download the files - without success. We don't have any download problems
anywhere else.

In other words, this is extremely frustrating. How can someone who starts using
this technology be successful?

Example:

[https://jaxb.dev.java.net/2.1.12/JAXB2_20090708.jar](https://jaxb.dev.java.net/2.1.12/JAXB2_20090708.jar) (10MBytes)
stopped at 376K etc, different sizes at different attempts, looks like a timeout.

I am suggesting you use a dial-up connection and see how far you get.

I feel that JAXB is a very nice technology and I am perplexed that developers
are faced with such unnecessary difficulties.
#### Environment
Operating System: All
Platform: All
#### Affected Versions
[2.1.12]
